PocketKit/Sources/PocketKit/MyList/ItemsList/SwiftUI/Components/OverflowMenu.swift

```diff
@@ -8,7 +8,7 @@ struct OverflowMenu: View {
     var trailingPadding: Bool = false
 
     var body: some View {
-        if overflowActions.isEmpty {
+        if !overflowActions.isEmpty {
```

Comment:
why are we reversing the check?

Reply:
it was a mistake from the other PR, we want to show the menu when it is not empty

Reply:
gotcha ok.